en/docs/get-started/configure-sms-otp-authenticator.md

```diff
@@ -0,0 +1,144 @@
+
+    ![sms_authentication_idp](../assets/img/get-started/quick-start-guide/sms-authentication-idp.png)
+
+3. Expand the **Federated Authenticators** > **SMS OTP Configuration** section.
+
+    ![expand_federated_authenticators](../assets/img/get-started/quick-start-guide/expand-federated-authenticators.png)
+
+4. Select both the **Enable** and **Default** checkboxes. This is to enable and make the **SMSAuthentication** authenticator the default one.
+
+    ![config_sms_otp](../assets/img/get-started/quick-start-guide/config-sms-otp.png)
+
+5. Based on your **SMS provider**, fill out the SMS OTP configurations.
+
+    - If Twilio is used as the SMS provider, go to [https://www.twilio.com/try-twilio](https://www.twilio.com/try-twilio) and create an account.
+
+    - While registering the account, verify your mobile number and click on console home [https://www.twilio.com/console](https://www.twilio.com/console)
+      to get free credits (Account SID and Auth Token).
+
+    - Twilio uses a POST method with headers and the text message and phone number are sent as the payload.
+
+6. Add the following sample configurations and click **Register**.
+
+     ``` tab="Format"
+     SMS URL : https://api.twilio.com/2010-04-01/Accounts/%7BAccountSID%7D/SMS/Messages.json
+     HTTP Headers : Authorization: Basic base64{AccountSID:AuthToken}
+     HTTP Payloads : Body=$ctx.msg&To=$ctx.num&From=urlencode{TrialNumber}
+     HTTP Method : POST
+     ```
+
+     ``` tab="Sample"
+     SMS URL : https://api.twilio.com/2010-04-01/Accounts/AC34f40df03e20fb6498b3fcee256ebd3b/SMS/Messages.json
+     HTTP Headers : Authorization: Basic QUMzNGY0MGRmMDNlMjBmYjY0OThiM2ZjZWUyNTZlYmQzYjo1ZmFkM2VkYzg4YWM1NTNiMmFiZjc4 NWI1MmM4MWFkYg==
+     HTTP Payloads : Body=$ctx.msg&To=$ctx.num&From=+1 210-880-1806
+     HTTP Method : POST
+     ```
+
+### Step 2: Configuring Account Lock
+
+1. Go to the **Main menu** > **Identity** > **Identity Providers** > **Resident** > **Login Attempt Security** > **Account Lock**.
+
+    ![select_resident_identity](../assets/img/get-started/quick-start-guide/go-to-resident-identity-providers.png)
+
+    ![security_account_lock](../assets/img/get-started/quick-start-guide/login-security-account-lock.png)
+
+2. Select the **Lock User Accounts** checkbox.
+
+    ![maximum_login_attempts](../assets/img/get-started/quick-start-guide/maximum-failed-login-attempts.png)
+
+3. Configure the **Maximum Failed Login Attempts**.
+
+4. Scroll down and click **Update**.
+
+### Step 3: Making Mobile Phone a Mandatory Claim
+
+1. Go to the **Main menu** > **Identity** > **Claims** > **List** > **http://wso2.org/claims** > **Mobile** > **Edit**.
+
+    ![select_claims_lists](../assets/img/get-started/quick-start-guide/go-to-claims-lists.png)
+
+    ![edit_mobile_claim](../assets/img/get-started/quick-start-guide/edit-mobile-claim.png)
+
+2. Select the **Required** checkbox.
```

Comment:
add a new step. after 1. 

2. Select `http://wso2.org/claims` from the list.
https://ob.docs.wso2.com/en/latest/assets/img/learn/identifier-first/select-claim-from-list.png